internal/providers/terraform/aws/registry.go

```diff
@@ -71,6 +71,9 @@ var ResourceRegistry []*schema.RegistryItem = []*schema.RegistryItem{
 	GetSNSTopicRegistryItem(),
 	GetSNSTopicSubscriptionRegistryItem(),
 	GetSQSQueueRegistryItem(),
+	GetNeptuneClusterRegistryItem(),
+	GetNeptuneClusterInstanceRegistryItem(),
+	GetNeptuneClusterSnapshotRegistryItem(),
 	GetNewEKSNodeGroupItem(),
```

Comment:
plz remember to search the Terraform resource list and anything that's not included here is probably free for this service

![Screen Shot 2021-06-16 at 9 22 51 AM](https://user-images.githubusercontent.com/638577/122184858-8b1ef980-ce84-11eb-98fc-4668ba07335a.png)